stream-chat-android-client/src/main/java/io/getstream/chat/android/client/notifications/ChatNotifications.kt

```diff
@@ -85,6 +86,14 @@ internal class ChatNotificationsImpl constructor(
         handler.onDismissNotification(notificationId)
     }
 
+    /**
+     * Dismiss notification associated to the [channelType] and [channelId] received on the params
```

Comment:
I think we should try to improve the documentation for our internal (and public) code. 

I think @zsmb13 prepared some guidelines on what they should look like, but generally, we should list out params using `@param`, we should make sure we finish all sentences with a full-stop (so that we know the comment is done) etc.

In general, we'll look back to these functions and having a bit more structure or description on how the function works will be helpful to internal devs too.

LMK what you think!